src/handlers/transformation/transform-csv-to-json.ts

```diff
@@ -0,0 +1,15 @@
+import { S3Event } from "aws-lambda";
+import * as AWS from "aws-sdk";
+import csv from "csvtojson";
+
+// Unsure how to test this function or indeed if it even needs testing given that it basically just using S3 and the csvtojson library. Feedback/Help welcome!
+export async function transformCsvToJson(event: S3Event): Promise<any[]> {
+  const S3 = new AWS.S3();
```

Comment:
Are we not usually using the AWS-SDK v3? The one with `s3-client` and `.send()`?

Reply:
Thanks - all done now!